flink-connector-hbase-base/src/test/java/org/apache/flink/connector/hbase/util/HBaseSerdeTest.java

```diff
@@ -99,7 +104,57 @@ void convertToReusedRowTest() {
                         "+I(2,+I(20),+I(Hello-2,200),+I(2.02,true,Welt-2))");
     }
 
-    private HBaseSerde createHBaseSerde() {
+    @Test
+    public void writeIgnoreNullValueTest() {
+        HBaseSerde serde = createHBaseSerde(false);
+        Put m1 = serde.createPutMutation(prepareRowData());
+        assert m1 != null;
+        Assert.assertEquals(10, Bytes.toInt(m1.getRow()));
```

Comment:
Pls. use `Assert4J`'s `assertThat` for all kinds of asserts. The community agreed that it should be the standard going forward as part of the [JUnit5 migration](https://docs.google.com/document/d/1514Wa_aNB9bJUen4xm5uiuXOooOJTtXqS_Jqk9KJitU/edit#heading=h.n2pt1kpwyals). It is not applied completely to this repo yet, but IMO it makes sense to conform with that if we touch some tests.